src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/JiraSoftwareScenario.kt

```diff
@@ -105,11 +103,11 @@ class JiraSoftwareScenario : Scenario {
             addComment to 2,
             browseProjects to 5,
             viewBoard to 10,
-            viewBacklog to 10,
+            workOnBacklog to 10,
```

Comment:
Thanks for updating the scenario too. It will give a richer coverage to the consumers. Also, it gets covered by `JiraSoftwareScenarioIT.shouldRunScenarioWithoutErrors`.